### PR TITLE
add klips logo and link to chart

### PR DESCRIPTION
**Pull request by sdobbert**
Adds the KLIPS-Logo to the chart and rounds the input data to 1 digit.

image

### CODE REVIEW

File: easy-to-use-api/klips-chart-api/src/util/Chart.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export const formatChartData = (data: TimeSeriesData): DataPointObject => {
         }
         result[key].push([
           obj.timestamp,
-          obj.band_1
+          parseFloat(obj.band_1).toFixed(1)
```
Comment (chrismayer) [inline easy-to-use-api/klips-chart-api/src/util/Chart.ts:99]: This is unrelated to the logo, right?

File: easy-to-use-api/klips-chart-api/src/util/Chart.ts [hunk 2]
```diff
@@ -158,6 +157,17 @@ export const setupBaseChart = (): echarts.EChartsOption => {
...
       bottom: 100
     },
+    graphic: [{
+      type: 'image',
+      onclick:  function () { window.open('http://www.klips-projekt.de/'); },
+      style: {
+        x: 8,
+        y: 20,
+        height: 40,
+        image: 'https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png',
```
Comment (chrismayer) [inline easy-to-use-api/klips-chart-api/src/util/Chart.ts:167]: For the moment OK to link the KLIPS logo of the main page. In the future we should consider hosting the logo our self, to avoid having another origin when embedding the widget.

**Review by chrismayer — APPROVED** [review]
Thanks, only 2 minor comments.

**Comment by sdobbert** [comment]
Added x-Axis configuration to adapt both x-Axes to input-Data. Also added a short error-message for wrong input for region in the URL.

**Comment by chrismayer** [comment]
> Added x-Axis configuration to adapt both x-Axes to input-Data. Also added a short error-message for wrong input for region in the URL.

@sdobbert Shouldn't this be better placed in a separate PR since this is a different thing? Or do I miss something?

**Comment by sdobbert** [comment]
Yes, thanks, moved to another pr.
